doc/install/fast-install-and-first-run.rst

```diff
@@ -94,7 +112,58 @@ Database init, data download & importation
    $ sudo ivre ipdata --download --import-all
 
 The two last step may take a long time to run (40 minutes on a decent
-server), nothing to worry about.
+server), nothing to worry about. To initiate flowcli, Neo4j default 
+password must be changed through its web interface. Connect to Neo4j
+IP address on port 7474 and enter neo4j:neo4j credentials to login
+then change the password. Don't forget to modify ``/etc/ivre.conf``
+accordingly.
+
+
+External programs
+-----------------
+
+- MongoDB Community Edition (latest version)
+- Nmap
+- Zeek/Bro
+- Argus
+- Nfdump
+- Neo4j
+
+::
+
+   $ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 9DA31620334BD75D9DCB49F368818C72E52529D4
+   $ echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.0.list
```

Comment:
Is this going to work on a Debian system?

Reply:
I confirm that it works in Debian stable "Buster". Note that the MongoDB official documentation is not up to date for Debian Buster.

Reply:
OK.